templates/sssd-session-recording.conf.epp

```diff
@@ -1,5 +1,6 @@
-<%- | $users,
+<%- | $exclude_users,
       $groups,
+      $users,
 | -%>
 # This file is managed by Puppet. Manual changes will be lost
 [session_recording]
```

Comment:
Should the if also check $exclude_users.empty ?

Reply:
I was referring to this line:
```
<% if ($users.empty) and ($groups.empty) { -%>
```

Reply:
Yeah, also updated the template so that it's more concise